### PR TITLE
Javascript/javascript2/week2

### PR DESCRIPTION
**Pull request by MaherHussain**

### CODE REVIEW

File: javascript/javascript2/week2/movies.js [hunk 1]
```diff
@@ -0,0 +1,320 @@
+const movies = [
```
Comment (flom84) [inline javascript/javascript2/week2/movies.js]: you should not make your own movies variable. Instead include javascript2/week2/homework/movies.js into your .html file

File: javascript/javascript2/week2/movies.js [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+// console.log(shortTitleMovies);
+
+
+// filtering as long title 
+const longTitleMovies = movies.filter((item) => item.title.length >= 10);
+// console.log(longTitleMovies);
+
+
+// filtering as year  
+const counter = movies.filter( item => item.year >= 1980 && item.year <= 1989
```
Comment (flom84) [inline javascript/javascript2/week2/movies.js]: Naming of the `counter ` variable is not okay....here . you return an array
```suggestion
const moviesFromEighties= movies.filter( item => item.year >= 1980 && item.year <= 1989 
```

File: javascript/javascript2/week2/movies.js [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+
+
+// filtering as year  
+const counter = movies.filter( item => item.year >= 1980 && item.year <= 1989 
+
+)
+console.log(counter.length);
+
+// adding tag prop to objects 
+const tagArray = movies.forEach(element => {
```
Comment (flom84) [inline javascript/javascript2/week2/movies.js]: Think of using `filter` method
```suggestion
const tagArray = movies.filter(element => {
```

File: javascript/javascript2/week2/movies.js [hunk 4]
```diff
@@ -0,0 +1,320 @@
...
+let word1 =  / surfer /g;
+let word2 = / the /g;
+let word3 = / 2 /g; ;
+
+
+console.log(
+  "the number of movies with surfer, the,hello is " +
+    movies
+      .filter((movies) =>
+        /surfer| the | hello /.test(movies.title.toLowerCase())
```
Comment (flom84) [inline javascript/javascript2/week2/movies.js]: I don't follow what you are trying here ? Are you using regular expressions

**Review by flom84 — CHANGES_REQUESTED** [review]

File: javascript/javascript2/week2/movies.js (hunk 4 above)
Comment (MaherHussain) [reply javascript/javascript2/week2/movies.js]: hey Florin. thanks. yes, I used regular expressions. 
I will try to make changes and write to you

**Pull request closed without merging**